### PR TITLE
ILS / LOC Fixes

### LINKED ISSUE #2560: KPAM
### Airport ICAO

KPAM

### Airports Issue Number

4370

### Runways with ILS/LOC

14L, 32R

### References

**HI-ILS OR LOC Y RWY 14L:** https://aeronav.faa.gov/d-tpp/2108/00312HILY14L.PDF
**HI-ILS OR LOC Y RWY 32R:** https://aeronav.faa.gov/d-tpp/2108/00312HILY32R.PDF

### Corrections Required

Correction of previous data.

### Are you working on this?

Yes

### Additional Information

_No response_

### LINKED ISSUE #2562: KPBG
### Airport ICAO

KPBG

### Airports Issue Number

7968

### Runways with ILS/LOC

35

### References

**ILS OR LOC RWY 35:** https://aeronav.faa.gov/d-tpp/2108/00729IL35.PDF

### Corrections Required

Correction of previous data.

### Are you working on this?

Yes

### Additional Information

_No response_

### LINKED ISSUE #2563: KPBI
### Airport ICAO

KPBI

### Airports Issue Number

182

### Runways with ILS/LOC

10L, 28R

### References

**ILS OR LOC RWY 10L:** https://aeronav.faa.gov/d-tpp/2108/00449IL10L.PDF
**ILS OR LOC RWY 28R:** https://aeronav.faa.gov/d-tpp/2108/00449IL28R.PDF

### Corrections Required

Correction of previous data.

### Are you working on this?

Yes

### Additional Information

_No response_

### LINKED ISSUE #2564: KPIR
### Airport ICAO

KPIR

### Airports Issue Number

3937

### Runways with ILS/LOC

31

### References

**ILS OR LOC RWY 31:** https://aeronav.faa.gov/d-tpp/2108/00324IL31.PDF

### Corrections Required

Correction of previous data.

### Are you working on this?

Yes

### Additional Information

_No response_

### LINKED ISSUE #2565: KPIT
### Airport ICAO

KPIT

### Airports Issue Number

826

### Runways with ILS/LOC

10L, 10R, 28L, 28R, 32

### References

**ILS RWY 10L (CAT II - III):** https://aeronav.faa.gov/d-tpp/2108/00570I10LC2_3.PDF
**ILS RWY 10R (CAT II - III):** https://aeronav.faa.gov/d-tpp/2108/00570I10RC2_3.PDF
**ILS OR LOC RWY 28L:** https://aeronav.faa.gov/d-tpp/2108/00570IL28L.PDF
**ILS RWY 28R (SA CAT I - II):** https://aeronav.faa.gov/d-tpp/2108/00570I28RSAC1_2.PDF
**ILS OR LOC RWY 32:** https://aeronav.faa.gov/d-tpp/2108/00570IL32.PDF

### Corrections Required

Correction of previous data.

### Are you working on this?

Yes

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by zhf21**
KPAM (Fixes #2560)
KPBG (Fixes #2562)
KPBI (Fixes #2563)
KPIR (Fixes #2564)
KPIT (Fixes #2565)

### CODE REVIEW

**Comment by zhf21** [comment]
Good to go. 👍

File: Glideslope.json [hunk 1]
```diff
@@ -21950,11 +21950,11 @@
...
-   "elevation": 17,
-   "latitude": 26.681817,
-   "longitude": -80.101708,
-   "bearing": 93.592,
-   "glideslope": 3.00092796,
+   "elevation": 19,
+   "latitude": 26.68321207,
+   "longitude": -80.108378,
+   "bearing": 92.8,
+   "glideslope": 3.00928,
```
Comment (velocity23) [inline Glideslope.json:21957]: Should be 3.000928 because the bearing is 092.8

**Review by velocity23 — CHANGES_REQUESTED** [review]

**Comment by zhf21** [comment]
@velocity23, I have fixed the GS for KPBI. The build failed, however. I don't believe it was due to my fix, though. I merged the remote-tracking branch since I do that before every edit I make, and if I read Travis's report correctly, he doesn't like the change with the template, which was included when I merged the remote-tracking branch. 

If you can take a look, I'd appreciate it.

File: Glideslope.json [hunk 2]
```diff
@@ -5,7 +5,6 @@
    "identifier": "IWG",
    "associatedRunwayNumber": "14L",
    "frequency": 110100,
-   "elevation": 129,
```
Comment (velocity23) [inline Glideslope.json:8]: Somehow this line has been removed, you'll need to re-add it. That's why the build is failing.

**Review by velocity23 — CHANGES_REQUESTED** [review]

**Review by velocity23 — APPROVED** [review]

**Pull request merged**